### DISCUSSION TITLE
How to continue reading from where I left?

### DISCUSSION THREAD

**Discussion by homocomputeris** [arianneorpilla/jidoujisho#290] [Q&A] [answered]
Every time I open any book, it opens on the cover. How do I continue reading from the page I stopped reading last time?

**Comment by arianneorpilla** [comment, marked as answer]
You can do this by bookmarking before you exit the book or turning on auto bookmark, see the [ttu repo](https://github.com/ttu-ttu/ebook-reader) for documentation.
